onchain/cairo/afk/Scarb.toml

```diff
@@ -34,6 +34,6 @@ url = "https://starknet-mainnet.public.blastapi.io/rpc/v0_7"
 block_id.number = "615925"
 
 [[tool.snforge.fork]]
-name = "Sepolia"
+name = "SEPOLIA_LATEST"
```

Comment:
Please do not change this, that's gonna broke all others tests.
Create your own fork to used in your test if you prefer